src/manager/index.html

```diff
@@ -16,7 +16,7 @@
     <body>
         <main class="tabbed loading">
             <nav class="topbar">
-                <ul class="tabstrip inline-list" role="tablist">
+                <ul class="tabstrip inline-list toolbar" role="tablist">
```

Comment:
I didn't have that set in the channel manager on purpose, though I guess the behavior is now different in the popup, so this would make them match. Which doesn't really resolve if I like this or not. Having a hover state is good, though I'm not sure if it'd be time to have a different active state for tabs then.

Reply:
An alternative is to add `button` elements to match the structures from the popup and options.

Reply:
I guess that'd make more sense. I'll do that after merging.